CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ This library adheres to [Semantic Versioning](https://semver.org/) and [Keep a C
 
 Nothing yet.
 
+## 2.0.2
```

Comment:
@dlh01 is it okay to make this update to a minor version? I'm not intimately familiar with all the changes from `alleyinteractive/wp-type-extensions` 2.0 to 3.0.

Reply:
I don't think there's anything in 3.0 that would specifically affect this library, although from the perspective of the library itself, I'm not sure that I'd label the change a "patch" version, though. Seems more like a minor version to me.

If you wanted to be really safe about it, I think you could do `^2.1|^3.0`, right?